### PR TITLE
TicTacToe implemented in react.js as assigned

### PR DESCRIPTION
**Pull request by Shivani-08**
**Description**
Implemented Tic-Tac-Toe Game in React.js.

**Issue**
#7 

**Requirements**

- Operating System : Windows
- IDE Used : WebStorm
- Language : JavaScript, CSS

**Screenshot of the Output**

reactttt

**Implementation**
There are two files in src folder:
- index.css: It is a CSS file used for styling of the web page.
- index.js: It is a JavaScript file in which the main react code has been written. There are two components in the file, namely, Board and Game. These components are responsible for the rendering and implementation of the game.

### CODE REVIEW

File: assignments/F-01-88/src/index.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+  ];
+  for (let i = 0; i < lines.length; i++) {
```
Comment (Jo-ravar) [inline assignments/F-01-88/src/index.js]: use ES6 map function instead

File: assignments/F-01-88/src/index.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+  jumpTo(step) {
+    this.setState({
+      stepNumber: step,
+      xIsNext: (step % 2) === 0
+    });
+  }
+
+  render() {
+    const history = this.state.history;
```
Comment (Jo-ravar) [inline assignments/F-01-88/src/index.js]: I think we should use all the calculation in a separate function . Lets use render to only return things.

Comment (Jo-ravar) [reply assignments/F-01-88/src/index.js]: @sinha24 what are the possible approach to do it in structured way?

File: assignments/F-01-88/src/index.js (hunk 1 above)
Comment (Shivani-08) [reply assignments/F-01-88/src/index.js]: Trying... will inform as soon as it is done.

**Comment by Shivani-08** [comment]
Added the comments.

**Comment by Shivani-08** [comment]
Okay, right now done from my side. I will try to do those changes as soon as i learn about it.

**Pull request merged**